apps/web/src/components/contributors.tsx

```diff
@@ -0,0 +1,130 @@
+			</div>
+		);
+	}
+
+	const sortedContributors = [...contributors].sort((a, b) => b.contributions - a.contributions);
+
+	return (
+		<div className="flex min-h-screen w-full flex-col pt-20">
+			<div className="flex-1 space-y-8 px-4 sm:px-6 lg:px-8">
+				<div className="space-y-4 text-center">
+					<h2 className="from-primary to-primary/60 bg-gradient-to-r bg-clip-text text-3xl font-bold tracking-tight text-transparent sm:text-4xl lg:text-5xl">
+						Our Contributors
+					</h2>
+					<p className="text-muted-foreground mx-auto max-w-3xl text-lg leading-relaxed sm:text-xl">
+						Meet the amazing people who make Nimbus possible
+					</p>
+				</div>
+
+				<div className="grid grid-cols-1 gap-6 sm:grid-cols-2 sm:gap-6 md:grid-cols-3 md:gap-7 lg:grid-cols-4 xl:grid-cols-5 2xl:grid-cols-6">
+					{sortedContributors.map(contributor => (
+						<Card
+							key={contributor.id}
+							className="group border-border/50 bg-card/50 hover:shadow-primary/20 relative flex aspect-square flex-col overflow-hidden shadow-lg backdrop-blur-sm transition-all duration-300 hover:-translate-y-2 hover:shadow-xl"
+						>
+							<div className="absolute inset-0 flex items-center justify-center opacity-5 transition-opacity duration-300 group-hover:opacity-10">
+								<Github className="text-foreground h-40 w-40" />
+							</div>
+
+							<div className="from-primary/10 absolute inset-0 bg-gradient-to-b via-transparent to-transparent opacity-0 transition-opacity duration-300 group-hover:opacity-100" />
+
+							<div className="relative z-10 flex h-full flex-col">
+								<div className="flex flex-1 flex-col items-center justify-center p-5 text-center">
+									<div className="relative mb-4">
+										<div className="from-primary/30 to-primary/10 absolute inset-0 rounded-full bg-gradient-to-r blur-md transition-all duration-300 group-hover:blur-lg" />
+										<Avatar className="ring-border group-hover:ring-primary/40 relative h-16 w-16 ring-2 transition-all duration-300">
+											<AvatarImage src={contributor.avatar_url} alt={contributor.login} />
+											<AvatarFallback className="from-primary/20 to-primary/10 text-primary-foreground bg-gradient-to-br text-sm font-bold">
+												{contributor.login.slice(0, 2).toUpperCase()}
+											</AvatarFallback>
+										</Avatar>
+									</div>
+
+									<div className="flex min-h-[4rem] flex-col justify-center">
+										<Link href={contributor.html_url}>
+											<CardTitle className="text-foreground hover:text-primary line-clamp-2 text-base leading-tight font-bold break-words hyphens-auto underline drop-shadow-sm transition-colors">
+												{contributor.login}
+											</CardTitle>
+										</Link>
+										<CardDescription className="text-muted-foreground mt-1 text-sm">Contributor</CardDescription>
+									</div>
+								</div>
+
+								<div className="flex flex-col items-center gap-3 p-4">
```

Comment:
With the 2 grid the contribution count gets hidden as the view box gets smaller. Can you make the card component responsive enough to display the count even on smaller screens? If not, we can hide it with this
```suggestion
								<div className="hidden flex-col items-center gap-3 p-4 sm:flex">
```

Reply:
I tried to make the card more responsive, now the contributions are visible on really small screens also.